static/js/zcommand.js

```diff
@@ -122,6 +166,18 @@ exports.process = function (message_content) {
         return true;
     }
 
+    const fluid_width_commands = ['/fluid', '/full'];
+    if (fluid_width_commands.includes(content)) {
+        exports.enter_fluid_mode();
+        return true;
+    }
+
+    const fixed_width_commands = ['/fixed', '/standard'];
```

Comment:
I don't think `/standard` is a good name to use here, since it's not clearly scoped to width.  Maybe just `fluid-width` and `fixed-width`?  With typeahead it should be fine.